src/entity.zig

```diff
@@ -225,34 +227,38 @@ pub const ClientEntityManager = struct {
 			}
 		}
 	}
-
 	pub fn serverUpdate(time: i16, data: []const u8) void {
```

Comment:
This function is called by the networking code. I think you should just pass in the network reader, and it's also valid to return an error.